docs/workflows/manage-content.rst

```diff
@@ -3,34 +3,41 @@
 Manage Docker Content in a Repository
 =====================================
 
-There are multiple ways that users can manage their Docker content in repositories:
+There are multiple ways that users can manage Docker content in repositories:
 
    1. :ref:`Tag<tagging-workflow>` or :ref:`Untag<untagging-workflow>` Manifests in a repository.
-   2. :ref:`Recursively add content<recursive-add>`
-   3. :ref:`Recursively remove content<recursive-remove>`
-   4. :ref:`Copy tags from source repository<tag-copy>`
-   5. :ref:`Copy manifests from source repository<manifest-copy>`
+   2. Recursively :ref:`add<recursive-add>` or :ref:`remove<recursive-remove>` Docker content.
+   3. Copy :ref:`tags<tag-copy>` or :ref:`manifests <manifest-copy>` from source repository.
 
 .. warning::
 
-   Users can also add and remove content using the `repository version create endpoint
+   Users **can but probably should not not** add and remove content
```

Comment:
s/content/docker content

Reply:
I think it is important to mention explicitly docker content, because it has relations. For example rpms would be completely ok with that endpoint.